package aufgabe_4;

public class MyException extends Exception
{
    public MyException() //Konstruktor unserer eigenen Methode
    {
        //Aufruf des Konstruktors mit neuem Fehlertext
        super("Da ist ein Fehler aufgetreten.");

    }




}
